### ISSUE TITLE
Clicking "OK" on read-only destination displays leaves the user stuck on in-progress page

### ISSUE THREAD

**Issue by shasheene** [rescuezilla/rescuezilla#363]
After clicking OK on an "read-only destination" error (ref: [Backup to read-only destination displays error box has confusing/unfriendly message (task #362)](https://github.com/rescuezilla/rescuezilla/issues/362)) we get stuck at 0% on the backup in-progress page:

![read-only](https://user-images.githubusercontent.com/1414379/184506140-bd1aaf6f-4afa-4ebd-9267-07b8da9096ea.png)
![stuck at 0%](https://user-images.githubusercontent.com/1414379/184506150-85dcfc3f-ab12-4e99-a876-994f93e5f23c.png)

The issue looks like it was introduced in Rescuezilla v2.2. It's just trying to close the internal log file but that hasn't been created yet.

```
AttributeError: 'NoneType' object has no attribute 'close'
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 153, in apport_excepthook
    with os.fdopen(os.open(pr_filename,
FileNotFoundError: [Errno 2] No such file or directory: '/var/crash/_usr_lib_python3_dist-packages_rescuezilla_rescuezilla.py.0.crash'

Original exception was:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/rescuezilla/backup_manager.py", line 1073, in completed_backup
    self.logger.close()
AttributeError: 'NoneType' object has no attribute 'close'
```

Two simple solutions:
1. Only close log file if it has been initialised
    * This will improve this specific issue
2. Improve handling when Exceptions occur during the summary page generation step
    * This will prevent similar unexpected events causing stuck at 0% situations.

Will be done for Rescuezilla v2.4.1.

**Comment by shasheene** [comment]
Keeping issue open until v2.4.1 released

**Comment by shasheene** [comment]
Fixed in [Rescuezilla v2.4.1](https://github.com/rescuezilla/rescuezilla/releases/latest)
